.github/merge_rules.json

```diff
@@ -15,42 +15,42 @@
          "torch/onnx/**"
       ],
       "approved_by": ["BowenBao", "garymm"],
-      "mandatory_app_id": 12274
+      "mandatory_checks_name": ["Facebook CLA Check"]
```

Comment:
Maybe at this point it'd be better to have a script to generate this file since I feel like editing json directly is going to begin to get a bit cumbersome here?